js/main.js

```diff
@@ -7,15 +7,7 @@ $(document).ready(function () {
   videoElement = video[0];
 
   $('.site-navbar').headroom();
-  $('p').widont();
-  $('h2').widont();
-  $('h3').widont();
-
-  // jQuery(function($) {
-  //   $('h1,h2,h3,li,p').each(function() {
-  //     $(this).html($(this).html().replace(/\s([^\s<]{0,10})\s*$/,'&nbsp;$1'));
-  //   });
-  // });
+  $('p,h2,h3,li').widont();
```

Comment:
I will rather prefer to use a class to handle this one, so every item that has the class we will apply the library, maybe "widont" class?